spec/controllers/application_controller_spec.rb

```diff
@@ -5,6 +5,17 @@
 require 'lib/sentry_logging_spec_helper'
 
 RSpec.describe ApplicationController, type: :controller do
+  before(:each) do
```

Comment:
Wow, i've never seen something quite like this in a controller spec. Not the fault of the Rubocop change -- just need to digest.